README.md

```diff
@@ -1,6 +1,6 @@
 # course-inventory
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/431390d7c73d47cfbf9654f0182ba6d6)](https://www.codacy.com/gh/tl-its-umich-edu/course-inventory)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/299102a0330d4dfd8e9f8d5ef36362b5)](https://www.codacy.com/manual/ITS_Teaching_And_Learning/course-inventory?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=tl-its-umich-edu/course-inventory&amp;utm_campaign=Badge_Grade)
 
```

Comment:
What are all these params you are adding after the Question mark? can you refer to the documentation that made you chose those options?

Reply:
Sorry, in Codacy, on the repository page under the General tab, there is a place where you can copy the Codacy Badge in Markdown. I got this from there.

Reply:
I could remove the parameters if you want?

Reply:
That's what codacy seems to suggest then you shouldn't be removing it.

Reply:
Based on what @lsloan had before, they probably aren't necessary and are primarily used for Codacy's own analytics. Not entirely sure though.

Reply:
I am ok either way, Looking at this [article](https://support.codacy.com/hc/en-us/articles/212799365-Badges) they are saying if you want to track code performance for the different branch you can use param for doing that. I don't feel the need to dig deep-down what all those params mean. I got curious about what those are and purpose